wandb/sdk/lib/runid.py

```diff
@@ -3,10 +3,12 @@
 runid util.
 """
 
+from typing import cast
+
 import shortuuid  # type: ignore
 
 
 def generate_id() -> str:
     # ~3t run ids (36**8)
     run_gen = shortuuid.ShortUUID(alphabet=list("0123456789abcdefghijklmnopqrstuvwxyz"))
-    return run_gen.random(8)
+    return cast(str, run_gen.random(8))
```

Comment:
Why do we need to do casting here? Looking at the code (and running local example) it seems that it returns `str`.

Reply:
shortuuid doesnt have type annotations.
We have a ticket to remove this dependency on this package, im hoping we get to it in this release.  this is needed to make typing work downstream until we replace this package